docs/source/duckdb_cli.md

```diff
@@ -0,0 +1,57 @@
+# DuckDB CLI
+
+The [DuckDB CLI](https://duckdb.org/docs/api/cli/overview.html) (Command Line Interface) is a single, dependency-free executable. 
+
+<Tip>
+
+For installation details, visit the [installation page](https://duckdb.org/docs/installation).
+
+</Tip>
+
+Starting from version `v0.10.3`, the DuckDB CLI includes native support for accessing datasets on Hugging Face via URLs. Here are some features you can leverage with this powerful tool:
+
+- Query public, gated and private datasets
+- Analyze datasets and perform SQL operations
+- Combine datasets and export it different formats
+- Conduct vector similarity search on embedding datasets
+- Implement full-text search on datasets
+- And more! For a complete list of DuckDB features, visit the DuckDB documentation.
```

Comment:
Not sure if this is the best link to link to, so feel free to replace with a more appropriate one!

```suggestion

For a complete list of DuckDB features, visit the DuckDB [documentation](https://duckdb.org/docs/).
```